### PR TITLE
chore: update docs

### PR DESCRIPTION
**Pull request by aouahib**

### CODE REVIEW

**Comment by aouahib** [comment]
Current dependencies on/for this PR:
* main
  * **PR #177** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/177" target="_blank">Graphite</a>  👈
    * **PR #178** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/178" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/177?utm_source=stack-comment).

**Review by andrewszucs — APPROVED** [review]

**Comment by aouahib** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/177?utm_source=stack-comment).


Job Reference: 7poIOCB6dCMmI0WRvE1R

**Pull request merged**
